src/main/java/com/commercetools/helper/mapper/impl/payment/BasePaymentMapperImpl.java

```diff
@@ -253,13 +254,21 @@ protected Details getTransactionDetails(@Nonnull CtpPaymentWithCart paymentWithC
      * @see #mapCustomLineItemToPaypalPlusItem(CustomLineItem, List)
      */
     protected Stream<Item> mapCustomLineItemToPaypalPlusItem(@Nonnull CustomLineItem customLineItem, @Nonnull List<Locale> locales) {
+        String itemName = customLineItem.getName().get(locales);
+        if (StringUtils.isBlank(itemName)) {
+            Locale fallBackLocale = customLineItem.getName().getLocales()
+                    .iterator().next();
+            itemName = customLineItem.getName().get(fallBackLocale);
+        }
+
         if (customLineItem.getDiscountedPricePerQuantity().size() > 0) {
+            String finalItemName = itemName;
```

Comment:
Why do you need `finalItemName` variable?

Reply:
It was added because of lambda expression reference. (final or effectively final variables can ONLY be used in a lambda expression) But now with the modification, we don't need it. anymore

Reply:
It seems you are using it 🤔

Reply:
I guess you were checking the outdated change. Here is the latest commit change. https://github.com/commercetools/commercetools-paypal-plus-integration/pull/166/files#diff-98208488d399ffeb7c0443fb763ae32aR256